README.md

```diff
@@ -50,6 +50,35 @@ hash.digest('hex');
 hash.digest('base64');
 ```
 
+## JavaScript ES6
+
+_Using the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API), available in all modern browsers. <sup>[[1]](https://caniuse.com/#feat=cryptography)</sup>_
+
+```js
+const key = 'the shared secret key here';
+const message = 'the message to hash here';
+
+const getUtf8Bytes = str =>
+  new Uint8Array(
+    [...unescape(encodeURIComponent(str))].map(c => c.charCodeAt(0))
```

Comment:
```suggestion
    decodeURI(encodeURIComponent(str)).map(c => c.charCodeAt(0))
```

you don't need the new array, right? and I hadn't heard of `unescape`, apparently it's not in the standard any more and is replaced with `decodeURI` - does it still work with this?

Reply:
That won't work for string inputs with unicode characters outside the ascii range (or something along those lines). 
`unescape(encodeURIComponent(...` is an old hack to turn a js string into a utf8 octet array http://monsur.hossa.in/2012/07/20/utf-8-in-javascript.html

The `TextEncoder` API would be ideal here but it's not supported in Edge yet unfortunately. 
`new TextEncoder().encode(str)`
https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder